src/runtimes/node/utils/zip.ts

```diff
@@ -53,6 +53,16 @@ interface ZipNodeParameters {
   srcFiles: string[]
 }
 
+const ensureBootstrapPresent = function (srcFiles: string[], aliases: Map<string, string>) {
```

Comment:
nit: A method starting with `ensure` makes me think that it'll check whether something exists and do something only if it doesn't (like `ensureDirectory`). That's not what's happening here, as we're always adding these files to the zip. I'd say a name like `addBootstrapFiles` would be more fitting.